samza-core/src/main/scala/org/apache/samza/checkpoint/OffsetManager.scala

```diff
@@ -412,8 +412,8 @@ class OffsetManager(
     val taskNameToSSPs: Map[TaskName, Set[SystemStreamPartition]] = systemStreamPartitions
 
     taskNameToSSPs.foreach {
-      case (taskName, systemStreamPartitions) => {
-        systemStreamPartitions.foreach { systemStreamPartition =>
+      case (taskName, systemStreamPartitionsSet) => {
```

Comment:
Let's revert this name change. The plural already indicates that this is a collection.